deployment_scripts/deploy_windows.ps1

```diff
@@ -1,16 +1,36 @@
-function Deploy-Windows([String] $monkey_home = (Get-Item -Path ".\").FullName, [String] $branch = "develop"){
+param(
+    [Parameter(Mandatory = $false, Position = 0)]
+    [String] $monkey_home = (Get-Item -Path ".\").FullName,
+
+    [Parameter(Mandatory = $false, Position = 1)]
+    [System.String]
+    $branch = "develop"
+)
+function Deploy-Windows([String] $monkey_home = (Get-Item -Path ".\").FullName, [String] $branch = "develop")
+{
+    Write-Output "Downloading to $monkey_home"
+    Write-Output "Branch $branch"
+    # Set variables for script execution
+    [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+    $webClient = New-Object System.Net.WebClient
+
+
     # Import the config variables
+    $config_filename = New-TemporaryFile
+    $config_filename = "config.ps1"
+    $config_url = "https://raw.githubusercontent.com/guardicore/monkey/" + $branch + "/deployment_scripts/config.ps1"
+    $webClient.DownloadFile($config_url, $config_filename)
     . ./config.ps1
     "Config variables from config.ps1 imported"
+    #Remove-Item $config_filename
+
```

Comment:
?

Reply:
Will revert. The config should be downloaded to a temp file then removed.